bitsong/assetlist.json

```diff
@@ -22,7 +27,358 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bitsong/images/btsg.png",
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bitsong/images/btsg.svg"
         },
+        "type_asset": "sdk.coin",
         "coingecko_id": "bitsong"
+      },
+      {
+        "description": "Adam Clay a BitSong Music FanToken",
+        "denom_units": [
+          {
+            "denom": "ft2D8E7041556CE93E1EFD66C07C45D551A6AAAE09",
+            "exponent": 0
+          },
+          {
+            "denom": "clay",
+            "exponent": 6
+          },
+          {
+            "denom": "ibc/7ABF696369EFB3387DF22B6A24204459FE5EFD010220E8E5618DC49DB877047B",
+            "exponent": 0,
+            "aliases": ["ft2D8E7041556CE93E1EFD66C07C45D551A6AAAE09"]
+          }
+        ],
+        "base": "ft2D8E7041556CE93E1EFD66C07C45D551A6AAAE09",
+        "name": "Adam Clay FanToken",
+        "display": "clay",
+        "symbol": "CLAY",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bitsong/images/ft2D8E7041556CE93E1EFD66C07C45D551A6AAAE09.png"
```

Comment:
not required, but I'd recommend considering renaming these to something more human readable.

Reply:
do you mean the logo_URIs? we prefer to set as denom, so it easily to get it